src/Repository/CommissionableAssetRepository.php

```diff
@@ -26,6 +26,20 @@ public function __construct(ManagerRegistry $registry)
         parent::__construct($registry, CommissionableAsset::class);
     }
 
+    /**
+     * @return CommissionableAsset[]
+     */
+    public function search(string $query): array
+    {
+        $words = explode(' ', $query);
```

Comment:
you can do a `preg_replace('/\s+/', ' ', trim($query));` before exploding the string